contributor-docs/versioning.md

```diff
@@ -0,0 +1,131 @@
+Primer you should feel confident that you can update without
+anything breaking in your project. For a full list of changes that the team
+commits to being backwards-compatible, view the [changes](#changes) table
+below.
+
+## Changes
+
+| Category   | Type of change                                                | semver bump |
+| :--------- | :------------------------------------------------------------ | :---------- |
+| Component  | A component is added                                          | `minor`     |
+|            | A component is deprecated                                     | `minor`     |
+|            | A component is removed                                        | `major`     |
+| Props      | A prop is added to a component                                | `minor`     |
+|            | The type of a prop is made more general                       | `minor`     |
+|            | The type of a prop is made more specific                      | `major`     |
+|            | A prop is deprecated                                          | `minor`     |
+|            | A prop is removed from a component                            | `major`     |
+|            | The element to which additional props are added to is changed |             |
+| Markup     | The DOM node that an `id` corresponds to is changed           |             |
+|            | The DOM node that an `aria-label` corresponds to is changed   |             |
+|            | The `role` of a component is changed                          |             |
+| Styles     | The `position` of the outermost element is updated            |             |
+|            | The `display` property of the outermost element is updated    |             |
+|            | A flex or grid property is updated                            |             |
+|            | A selector is added                                           |             |
+|            | A selector is removed                                         |             |
+|            | The specificity of a selector is raised                       |             |
+|            | The specificity of a selector is lowered                      |             |
+| Behavior   | Interactions are added to a component                         |             |
+|            | Interactions are removed from a component                     |             |
+| TypeScript | A type is added                                               | `minor`     |
+|            | A type is made more general                                   | `minor`     |
+|            | A type is made more specific                                  | `major`     |
+|            | A type is removed                                             | `major`     |
+| Package    | An entrypoint is added to the package                         | `minor`     |
+|            | An entrypoint is removed from a package                       | `major`     |
+
```

Comment:
Should we add a section for `Accessibility` or things related to accessibility can go under other corresponding sections?

Reply:
@broccolinisoup makes sense to me! What kind of situations would be good to place under an `Accessibility` category?

Reply:
I guess things in my mind can certainly go under other categories but maybe explicitly having a section for `Accessibility` feels more inclusive? 🙂 

I was thinking;
- Exposing a mandatory `aria` attribute in the API
- Introducing a11y remediations that result in breaking changes in a current user behaviour
- Updating keyboard interactions

Does any of it sound applicable at all?